include/static-defines.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
```

Comment:
you're doing something wrong if you need to define these, I can't merge this

Reply:
File should probably be removed altogether

Reply:
Disagree, mod won't compile without this because of references to ID

Reply:
then something is going wrong, as these defines really are not needed if compilation is setup right

Reply:
By all means, try to compile it without that include. If you can get it to work then I'd like to know the steps as for me I had to add this line to get it to compile.